parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -88,6 +88,22 @@ write_files:
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "kubeletSystemdService"}}
 
+{{if not .IsVHDDistro}}
```

Comment:
nit:
{{- if not .IsVHDDistro}}
 this preserves formatting and removes unnecessary whitespaces in template generation

Reply:
Done!